### PR TITLE
Cleanup QRegExp includes

### PR DESCRIPTION
**Pull request by sgaist**
This patch adds the missing QRegExp includes and removes them when
not used in prevision of the qtbase include cleanup.

### CODE REVIEW

File: Tools/DumpRenderTree/qt/DumpRenderTreeQt.cpp [hunk 1]
```diff
@@ -48,6 +48,7 @@
 
 #include <QApplication>
 #include <QBuffer>
+#include <QRegExp>
```
Comment (annulen) [inline Tools/DumpRenderTree/qt/DumpRenderTreeQt.cpp]: Alphabetic order is broken here, as you can see in Travis CI log (we run style checker there:)

**Comment by annulen** [comment]
BTW, I think it would be better to port uses of QRegExp in QtTestBrowser, MiniBrowser, and DumpRenderTree to QRegularExpression

File: Tools/DumpRenderTree/qt/DumpRenderTreeQt.cpp (hunk 1 above)
Comment (sgaist) [inline Tools/DumpRenderTree/qt/DumpRenderTreeQt.cpp]: Good thing, I missed the alphabetical sorting done.

By the way, shouldn't I have got an email from Travis about that ?

**Comment by sgaist** [comment]
It's on the back burner. I'm currently doing the same cleanup in Qt and I'm slowly porting things to `QRegularExpresssion` but I have one commit for Qt base that can trigger a small SIC in the sense that some code is using an indirect inclusion of `QRegExp`through `QStringList` so I wanted to avoid you the need to trigger a rebuild of your packages just for that while providing an update to QRegularExpression later.

**Comment by annulen** [comment]
Thanks!

**Pull request merged**